terraform/templates/terraform.tfplan.gotmpl

```diff
@@ -5,7 +5,7 @@
 <details><summary>Expand to view details</summary><p>
 
 ```
-{{file.Read "/temp/tfplan.txt"}}
```

Comment:
@kobrikx why do we need to make changes there?

Reply:
Otherwise it can't find the file. This folder is mounted to Gomplate in docker.